common/src/main/java/org/wso2/testgrid/common/config/JobConfigYaml.java

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
For the class name, IMO `JobConfig` is enough. We do not need to mention the file type we are using... after sometime it could be some different file format

Reply:
Agreed with Vidura :)